_appmap/recording.py

```diff
@@ -81,6 +81,23 @@ def __exit__(self, exc_type, exc_value, tb):
         return False
 
 
+def check_call_return_stack_order(events):
```

Comment:
We don't want to do this in the agent. AppMaps can be quite large, and this will have a significant performance impact. Detecting the problem during fingerprinting is good enough

Reply:
I removed this commit.

(My reasoning was that it would have an overhead but this overhead would always go with the main overhead of creating large AppMaps and it would be small compared to the large AppMap creation itself.)